### DISCUSSION TITLE
[Mod] exact BPM change no matter the maps original BPM

### DISCUSSION THREAD

**Discussion by zabbazebra** [ppy/osu#30123] [Ideas] [answered]
Before you ask, i know double time RATE CHANGE exists! What i will describe is slightly different.

A mod that allows the player to select a singular bpm and, no matter what map they play, the map will be that bpm.

Example: I set the bpm to 250 and play "Lonely Go" which is 200bpm. (This is the rate change equivalent of 1.25). The map will be in 250 bpm. I can then exit lonely go and enter any other map (WITHOUT CHANGING MY MOD SETTINGS) like "Horrible Kids" and it will ALSO be 250 bpm, even though the original bpm is 170 bpm. (This is the rate change equivalent of 1.47x). I could then go and play "Gotta Go Fast (TV Size)" which IS ORIGINALLY 250 bpm and it would literally just be nomod.

Keep in mind, instead of being added as a whole new mod, this could be added onto double time as we know it as something like "automatic bpm change" selection in the mod customization section. However, allowing players to change the bpm WITHOUT increasing Approach rate and Overall difficulty would add another layer of customization. That could also be done with "dont scale approach rate" and "dont scale overall difficulty" selections in the mod customization section.

When you want to practice a certain bpm, whether it be jumps or streams or bursts, you would have to change your rate change amount per map in order to get that bpm. This would add some quality of life by making practice sessions more efficient and less tedious, allowing us to jump between any maps we like.

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/22616, please upvote that!
